DesktopEdge/Views/Screens/MainMenu.xaml.cs

```diff
@@ -226,40 +224,40 @@ private void UpdateState() {
                 VersionInfo.Content = $"App: {appVersion} Service: {version}";
 
             } else if (menuState == "Advanced") {
-                MenuTitle.Content = "Advanced Settings";
+                MenuTitle.Content = "ADVANCED SETTINGS";

```

Comment:
Did not have to cap-all each of these, just did for a better code-understanding that it would look this way.  An add-in to a later function that assigns the title to the appropriate screen auto-caps the title regardless so there is never a situation where the title could look different across screen.s